docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -3,82 +3,90 @@ include::./attributes.adoc[]
 
 == Gradle configuration
 
-Configure Gradle as indicated in the link:gradle-config.html[Gradle configuration page].
+Configure your project as indicated in the link:gradle-config.html[Gradle configuration page].
 
-At the moment there is no way of automatically generating a new project using the {project-name} Gradle plugin, 
+At the moment there is no way of automatically generating a new project using the {project-name} Gradle plugin,
 luckily setting up a {project-name} project with Gradle is very simple. You only need to add the {project-name} Gradle plugin like this:
 
 [source,groovy,subs=attributes+]
 ----
-apply plugin: 'io.quarkus.gradle.plugin'
+plugins {
+    id 'java'
+    id 'io.quarkus.gradle.plugin' version '{quarkus-version}'
+}
 ----
-Note: If you did not follow the steps indicated in the link:gradle-config.html[Gradle configuration page]
-you need to add this block to your 'build.gradle' file as well:
 
-[source,groovy,subs=attributes+]
+or, if you use the Gradle Kotlin DSL:
+
+[source,kotlin,subs=attributes+]
 ----
-buildscript {
-  repositories {
-    mavenCentral()
-  }
-  dependencies {
-    classpath 'io.quarkus:quarkus-gradle-plugin:{quarkus-version}'
-  }
+plugins {
+    java
+    id("io.quarkus.gradle.plugin") version "{quarkus-version}"
 }
 ----
 
 [[project-creation]]
 == Creating a new project
 
-For now we have to manually create a Gradle project file for {project-name}. 
+For now we have to manually create a Gradle project file for {project-name}.
 Here is a complete sample file for a simple rest project:
 
 [source,groovy,subs=attributes+]
 ----
-apply plugin: 'java'
-apply plugin: 'io.quarkus.gradle.plugin' <1>
-
-
-buildscript { <2>
+plugins {
+    id 'java'
+    id 'io.quarkus.gradle.plugin' version '{quarkus-version}' <1>
+}
 
-  repositories {
+repositories {
     mavenCentral()
+
+    // this is temporary, all dependencies should be in central soon
     maven {
-      //this is temporary, all dependencies should be in central soon
-      url 'http://repository.jboss.org/nexus/content/groups/public'
+        url = 'http://repository.jboss.org/nexus/content/groups/public'
     }
-  }
-  dependencies {
-    classpath 'io.quarkus:quarkus-gradle-plugin:{quarkus-version}'
-  }
-}
-
-repositories { <3>
-  mavenCentral()
-  maven {
-    url 'http://repository.jboss.org/nexus/content/groups/public'
-  }
 }
 
-dependencies { <4>
-  compileOnly group: 'io.quarkus', name: 'quarkus-resteasy', version:'{quarkus-version}'
-  testOnly group: 'io.quarkus', name: 'quarkus-junit5', version:'{quarkus-version}'
-  testCompile group: 'io.rest-assured', name: 'rest-assured', version: '3.3.0'
+dependencies { <2>
+    compileOnly 'io.quarkus:quarkus-resteasy:{quarkus-version}'
+    compileOnly 'io.quarkus:quarkus-arc:{quarkus-version}'
```

Comment:
I think quarkus-arc is transitively implicit? It works for me without this over in https://github.com/quarkusio/quarkus-quickstarts/issues/75 (see https://github.com/quarkusio/quarkus-quickstarts/compare/master...vorburger:gradle)

Reply:
If it's confirmed that it's not needed, then the getting started guide using Maven (https://quarkus.io/guides/getting-started-guide) (which I used as a basis for this gradle example) should probably be modified, too: it adds the arc dependency explicitly.

Reply:
@jnizet it's not strictly required because indeed transitive (just verified), but I just noticed that it's everywhere (also in all quickstarts), so I guess the idea is that it's clearer to have it explicit (perhaps to illustrate that you could ditch quarkus-resteasy and still use just quarkus-arc); therefore please ignore this point and keep it here as well - sorry for the confusion.

Reply:
@jnizet I confirm the arc dependency is not necessary. I'll fix the Maven getting started guide.

Reply:
@gsmet it's also [in all the quickstarts](https://github.com/quarkusio/quarkus-quickstarts/search?utf8=%E2%9C%93&q=quarkus-arc&type=) ...